MIGRATION.md

```diff
@@ -0,0 +1,82 @@
+Migrating to @oclif/core
+==============
+
+Migrating to `@oclif/core` from the old oclif libraries (`@oclif/config`, `@oclif/command`, `@oclif/error`, `@oclif/parser`) is relatively straight forward.
+
+## Update Imports
+
+Any imports from the old libraries can be replaced with `@oclif/core`. For example,
+
+```typescript
+import Help from '@oclif/plugin-help';
+import {Topic} from '@oclif/config';
+import {Command, flags} from '@oclif/command'
+```
+
+Can be changed to:
+
+```typescript
+import {Command, flags, Topic, Help} from '@oclif/core';
+```
+
+## Update your bin scirpts
+
+`@oclif/core` now supports separate bin scripts for production and development.
+
+These new bin scripts can be copied directly from our [example repository](https://github.com/oclif/hello-world/tree/main/bin).
+
+## Restore `-h`, `-v`, and `version`
+
+`@oclif/config` automatically added `-h` as a short flag for `--help`, `-v` as a short flag for `--version`, and `version` as an alias for `--version`.
+
+`@oclif/core` removes these so that you're able to use those flags for whatever you want! However, we've added a way to restore that functionality if you would like to keep it.
```

Comment:
```suggestion
`@oclif/core` removes these so that you can now use those flags for whatever you want! However, we've added a way to restore that functionality if you want to keep it.
```